src/rtp.c

```diff
@@ -1218,7 +1218,8 @@ gboolean janus_rtp_simulcasting_context_process_rtp(janus_rtp_simulcasting_conte
 		context->last_relayed = now;
 	} else if(context->substream > 0) {
 		/* Check if too much time went by with no packet relayed */
-		if((now - context->last_relayed) > (context->drop_trigger ? context->drop_trigger : 250000)) {
+		gint64 delayMicrosec = (now - context->last_relayed);
```

Comment:
The variable name is still camel case.

Reply:
Fixed and aligned the name with 0.x (which was ok - delay_us)
Sorry about this